cluster/manager.go

```diff
@@ -50,6 +52,13 @@ func NewManager(clusters clusterRepository, secrets secretValidator, events clus
 		clusters: clusters,
 		secrets:  secrets,
 		events:   events,
+		statusChangeDuration: prometheus.NewSummaryVec(prometheus.SummaryOpts{
```

Comment:
Create the collector outside and inject it into the manager.

Reply:
fixed